### ISSUE TITLE
Can't resolve 'fs' from debug package

### ISSUE THREAD

**Issue by darkadept** [jcoreio/crater#168]
This isn't an issue with the current code base bit I'm running into it with my additions to Crater.

I've taken what I could from the webpack 2 branch and tried integrating stuff from the master branch plus upgrading package versions among other things.

This error happened when I incorporated the "fix(modules) only resolve fibers from bundle.." changes.

Error message on server:
```
ERROR in ./build/meteor/bundle/programs/server/npm/~/debug/node.js
Module not found: Error: Can't resolve 'fs' in '/home/mike/dev/thr3/build/meteor/bundle/programs/server/npm/node_modules/debug'
 @ ./build/meteor/bundle/programs/server/npm/~/debug/node.js 163:15-28
 @ ./src/core/client/index.js
 @ multi ./src/core/client/index.js react-hot-loader/patch webpack-hot-middleware/client

ERROR in ./build/meteor/bundle/programs/server/npm/~/debug/node.js
Module not found: Error: Can't resolve 'net' in '/home/mike/dev/thr3/build/meteor/bundle/programs/server/npm/node_modules/debug'
 @ ./build/meteor/bundle/programs/server/npm/~/debug/node.js 170:16-30
 @ ./src/core/client/index.js
 @ multi ./src/core/client/index.js react-hot-loader/patch webpack-hot-middleware/client
```

I recall mention of something specific to the "debug" package. Is this the same problem?

**Comment by jedwards1211** [comment]
Probably you have a mix of webpack v1-related tools with the webpack 2 stuff.  It's way past time to merge the webpack2 branch into master, I'll try to do that soon.  The only thing that had been holding me up was a code coverage drop that I was having a devil of a time figuring out.

**Comment by darkadept** [comment]
Yeah it's possible. I'll definitely help out if you start merging webpack2 back into master. I'm just not quite sure where you'd start?

**Comment by jedwards1211** [comment]
Yeah, I'm trying to get there, fixing up a few other things first.  Sorry it's taking me long, sometimes it's really horrible trying to get the CI build to work after I change anything, because there are so many damn tools at play.
